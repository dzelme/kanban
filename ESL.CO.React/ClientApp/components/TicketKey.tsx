import * as React from 'react';

export default class TicketKey extends React.Component<{ keyName: string }> {
    public render() {

        return <div>
            <h4><strong>{this.props.keyName}</strong></h4>
        </div>

    }
}