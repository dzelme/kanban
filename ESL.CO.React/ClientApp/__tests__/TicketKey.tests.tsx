// Link.react.test.js
import * as React from 'react';
import TicketKey from '../components/TicketKey';
import * as renderer from 'react-test-renderer';

test('Renders keyname prop', () => {
    const component = renderer.create(
        <TicketKey keyName="123" />,
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});