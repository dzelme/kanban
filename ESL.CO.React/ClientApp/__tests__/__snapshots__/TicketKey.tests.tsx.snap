// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<div
  className="TicketText"
>
  <h5>
    123
  </h5>
</div>
`;
